### DISCUSSION TITLE
wie installiere ich die Go E Charger Repo

### DISCUSSION THREAD

**Discussion by disclaimer01** [cathiele/homeassistant-goecharger#66] [Q&A]
Hallo, 
ich habe über HACS das Repo geladen und konnte es dann auch im Add Store finden. Wenn ich es anklicke kommt folgendes Fenster:

![grafik](https://user-images.githubusercontent.com/97065450/166135397-272f1ed1-08c6-4eae-85c8-4a27b8b9ec8b.png)

Was soll dieser Fehler bedeuten?
Muss ich wie in der ANleitung noch das Git kopieren und dann in das Verzeichnis verschieben? Ich nutze Home Assistent auf einem Pi 4  als fertiges Image welches ich vom Pi Imager bezogen habe.

![grafik](https://user-images.githubusercontent.com/97065450/166135492-010bf1be-f9e5-45e0-b189-22874f07e471.png)

Danke für eure Unterstützung

**Comment by cathiele** [comment]
Die Installation ist so schon richtig. bitte mal versuchen bei dem korrekturfaktor 1.00 einzugeben. Hintergrund:

Die Messung der Verbräuche im charger ist zumindest bei frühen Versionen sehr ungenau. Daher kann hier ein Faktor angegeben werden. 1.00 bedeutet, dass die Werte für die korrigierten Attribute denen der Rohdaten entsprechen. 1.10 bedeutet also, dass der gemessene Wert nach oben korrigiert wird. Am besten hierzu die Spannung am Charger messen und mit dem im Charger angezeigten Wert vergleichen.

Leider ist der Onboarding-Prozess bei der Integration dieser Funktionalität etwas kaputt gegangen. Daher kann aktuell scheinbar die Einrichtung so nicht erfolgen. Kannst du bitte mal mit 1.00 als Wert eingeben und testen ob es damit geht?

**Reply by disclaimer01** [reply]
ok. Wie bzw. wo ändere ich den Wert? oder Repo löschen und neu installieren?

**Comment by disclaimer01** [comment]
Moin, ich habe es wieder versucht die go-e in Home Assistent einzubinden. Ich nutze die HomeAssistent Version die man vom Pi Imager installiert bekommt. Grundfunktionen wie Temperaturfühler und Steckdosen konnte ich auch ziemlich leicht installieren. Bei der go-e sieht es etwas anders aus. Ich habe die über Hacs installieren können (Repo heruntergeladen Verzeichnisse kopiert...) und über Integrationen dann auch installiert:
![grafik](https://user-images.githubusercontent.com/97065450/190860684-c5d9fb17-d0fc-437e-8b8c-bb307a410df0.png)
Demnach findet er ja auch das Gerät und die Entitäten. Nun kann ich aber keine Konfiguration durchführen:
![grafik](https://user-images.githubusercontent.com/97065450/190860726-30fadb36-43e3-4dbd-a0a7-c10bba6dc918.png)

Auch ein erneutes Laden ändert nichts. 
Ich habe in den Einstellungen der App
die Cloud Verbindung aktiviert
lokale HTTP API v2 ist aktiviert
lokale HTTP API v1 ist aktiviert.

Ich möchte ja auch keine Ladung damit steuern, das läuft alles über eine App Namens Wallbox-Steuerung. Die Steuert super die Überschussregelung.
Ich möchte nur den Stromverbrauch sehen, wo welcher Strom im Haus verbraucht wird. Und da fehlt mir halt die go-e.  Das andere ist alles ziemlich klar für mich, Großgeräte laufen über Steckdosen die ich auswerten/sehen kann. Und auf den Herd kann ich in der Übersicht verzichten.

**Reply by cathiele** [reply]
Da war noch ein bug im code. Mit Version 0.25.2 sollte der DIalog nun funktionieren. v1 API wird benötigt.

**Reply by gzebedin** [reply]
I am getting the same problem with the newest HA.
![Screenshot 2024-02-19 091017](https://github.com/cathiele/homeassistant-goecharger/assets/44834508/0b1f7239-5961-49d4-83d1-335c51e8459e)
Does anybody have an idea? I activated v1 API with the app.

**Comment by disclaimer01** [comment]
Moin,
Ja kann ich machen wenn du mir sagst wo und wie. ...
Ich bin in diese Sachen nicht sehr unbeholfen. Ich kann auch nicht sagen wo die Dateien auf meinem Pi liegen da ich ein fertiges Image vom Imager nutze.

Gruß Ulf

Am 1. Mai 2022 19:11:05 MESZ schrieb Carsten Thiele ***@***.***>:
>Die Installation ist so schon richtig. bitte mal versuchen bei dem korrekturfaktor 1.00 einzugeben. Hintergrund:
>
>
>
>Die Messung der Verbräuche im charger ist zumindest bei frühen Versionen sehr ungenau. Daher kann hier ein Faktor angegeben werden. 1.00 bedeutet, dass die Werte für die korrigierten Attribute denen der Rohdaten entsprechen. 1.10 bedeutet also, dass der gemessene Wert nach oben korrigiert wird. Am besten hierzu die Spannung am Charger messen und mit dem im Charger angezeigten Wert vergleichen.
>
>
>
>Leider ist der Onboarding-Prozess bei der Integration dieser Funktionalität etwas kaputt gegangen. Daher kann aktuell scheinbar die Einrichtung so nicht erfolgen. Kannst du bitte mal mit 1.00 als Wert eingeben und testen ob es damit geht?
>
>
>
>-- >
>Reply to this email directly or view it on GitHub:
>
>https://github.com/cathiele/homeassistant-goecharger/discussions/66#discussioncomment-2669838
>
>You are receiving this because you authored the thread.
>
>
>
>Message ID: ***@***.***>
Höfs, Windbergen
